api/admin/controller/library_settings.py

```diff
@@ -337,18 +331,37 @@ def list_setting(self, setting, json_objects=False):
 
         return json.dumps([_f for _f in value if _f])
 
-    def image_setting(self, setting):
-        """Retrieve an uploaded image for the given setting from the current HTTP request."""
+    @staticmethod
+    def _data_url_for_image(image: Image, _format="PNG") -> Optional[str]:
+        """Produce the `data` URL for the setting's uploaded image file.
+
+        :param image: A Pillow Image object.
+        :param _format: A valid Pillow image format.
+        :return: The `data` URL if an image file was uploaded, else None.
```

Comment:
This might be a copy paste error from below? This doesn't look like it will ever return `None` to me.

Reply:
Absolutely! Thanks!